tile.py

```diff
@@ -6,9 +6,10 @@
 
 
 class Tile:
-    def __init__(self, direction, name, properties):
+    def __init__(self, direction, name, type, properties):
```

Comment:
Před časem jsem používala "type" jako název proměnné v testech. Pak jsme se domluvili, že to není nejvhodnější způsob, protože type je zároveň slovo s pthonním významem. Pro jednoznčnost navrhuji něco jako `tile_type`.

Reply:
Aha, to máš pravdu. Díky!